_posts/fr/newsletters/2024-08-23-newsletter.md

```diff
@@ -0,0 +1,239 @@
+---
+title: 'Bulletin Hebdomadaire Bitcoin Optech #317'
+permalink: /fr/newsletters/2024/08/23/
+name: 2024-08-23-newsletter-fr
+slug: 2024-08-23-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+Le bulletin de cette semaine résume une discussion sur un protocole anti-exfiltration qui ne
+nécessite qu'un seul aller-retour de communication entre un portefeuille et un dispositif de
+signature. Sont également incluses nos sections
+régulières décrivant les mises à jour des clients et services, avec les annonces de nouvelles versions et les changements
+apportés aux principaux logiciels d'infrastructure Bitcoin.
+
+## Nouvelles
+
+- **Protocole anti-exfiltration simple (mais imparfait) :** le développeur Moonsettler a
+  [posté][moonsettler exfil1] sur Delving Bitcoin pour décrire un protocole [anti-exfiltration][topic
+  exfiltration-resistant signing]. Le même protocole a été décrit auparavant (voir les Bulletins
+  [#87][news87 exfil] et [#88][news88 exfil]), avec Pieter Wuille [citant][wuille exfil1] la
+  description la plus ancienne connue de la technique pour l'anti-exfil étant un [post de
+  2014][maxwell exfil] par Gregory Maxwell.
+
+  Le protocole utilise le protocole sign-to-contract pour permettre à un portefeuille logiciel de
+  contribuer à l'entropie sélectionnée par un dispositif de signature matériel d'une manière qui
+  permet au portefeuille logiciel de vérifier plus tard que l'entropie a été utilisée.
+  Sign-to-contract est une variation de [pay-to-contract][topic p2c] : dans pay-to-contract, la clé
```

Comment:
publique du destinataire est modifiée ; dans sign-to-contract, le nonce de signature de l'envoyeur